src/Table/index.tsx

```diff
@@ -25,5 +27,24 @@ export const StyledTable = styled(AntdTable)`
 export function Table<
   RecordType extends Record<string, unknown> = Record<string, unknown>,
 >(props: TableProps<RecordType>) {
-  return <StyledTable rowKey="id" {...props} />;
```

Comment:
@mic-web FYI: another possibility is to wrap the styled tabel in a Spin component like I I did it for the `microplate`-component:
`
    <Spin
      spinning={props.loading ?? false}
      indicator={
        <MllSpinnerIcon
          style={{
            width: '2em',
          }}
        />
      }
    >
    `